packages/front-end/components/Experiment/MetricsOverridesSelector.tsx

```diff
@@ -436,8 +438,8 @@ export default function MetricsOverridesSelector({
               options={unusedMetrics.map((m) => {
                 const metric = metricDefinitions.find((md) => md.id === m);
                 return {
-                  label: metric.name,
-                  value: metric.id,
+                  label: metric?.name,
+                  value: metric?.id,
```

Comment:
Can we just use `m` here instead of `metric?.id`?  That way, it will always have a proper value at least.  Same for label, should we fall back to something like `"Unknown Metric"`?